### DISCUSSION TITLE
[Projects Beta]: Reorder views

### DISCUSSION THREAD

**Discussion by mtruongle** [community/community#7455] [Projects and Issues] [answered]
Once created, the views cannot be moved anymore. It would be really nice if we can reorder views.

**Comment by jamemackson** [comment]
yes, this feels like a must have!

**Comment by ls-jonathan-jouret** [comment]
Must have for me too! Nice work on the new projects btw, I love it

**Comment by Katsute** [comment, marked as answer]
https://github.com/github/feedback/discussions/4803

**Comment by azenMatt** [comment]
Hi everyone - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄

**Comment by yringler** [comment]
I was able to reorder, but didn't know it was working untill I let go of the mouse - there isn't much of a visual indicator that it's moving.
I see now there's that blue line thingie, but that's pretty subtle.
I expect to see it actually move, and the other things shift. That's clear.

**Comment by bcb** [comment]
The reordering works on desktop, but doesn't work on mobile in either table or board view.
